packages/core/src/sagas/sendMessageBackToPostActivitySaga.js

```diff
@@ -0,0 +1,30 @@
+import {
+    put,
+    take
+  } from 'redux-saga/effects';
+
+  import whileConnected from './effects/whileConnected';
+
+  import { SEND_MESSAGE_BACK } from '../actions/sendMessageBack';
+  import postActivity from '../actions/postActivity';
+
+  export default function* () {
+    yield whileConnected(function* () {
+      for (;;) {
+        const { payload: { displayText, text, value } } = yield take(SEND_MESSAGE_BACK);
```

Comment:
Instead of `take`, should use `takeEvery`.

Take a look at [`sendMessageToPostActivitySaga.js`](https://github.com/Microsoft/BotFramework-WebChat/blob/master/packages/core/src/sagas/sendMessageToPostActivitySaga.js).